<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fftMIDI</class>
 <widget class="QMainWindow" name="fftMIDI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider_time_quantum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_activation_level">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_quantum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_activation_level">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Cutoff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time_quantum_display">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0 ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_activation_level_display">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_infile_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_outfile_text">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_infile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File (WAV)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_outfile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File (MIDI)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_run">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_run_msg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
